CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### Next
+- Added option to specify profile when connecting to Redshift via IAM
+
+
```

Comment:
Instead of adding this `### Next` section at the top, can you add a `### Features` section under the `## dbt 0.18.0 (Release TBD)` line, and add your line there? Please also add a link to the issue/PR, like the other features listed!

Also, please add a new contributors section to the `## dbt 0.18.0 (Release TBD)` section with your username and a link here. Check out lower down in the file for the exact formatting!

Reply:
Done!